app/src/main/java/org/mozilla/focus/activity/EditBookmarkActivity.kt

```diff
@@ -27,12 +27,41 @@ class EditBookmarkActivity : AppCompatActivity() {
     private val bookmark: TempInMemoryBookmarkRepository.Bookmark by lazy { TempInMemoryBookmarkRepository.getInstance().get(itemId) }
     private val name: EditText by lazy { findViewById<EditText>(R.id.bookmark_name) }
```

Comment:
I feel this should be renamed to something that can self-explain it's a UI widget, so one can easily tell it's not something parallel to another member variable "originalName", which is of type String.